core/src/sql/statements/define/table.rs

```diff
@@ -30,6 +30,8 @@ pub struct DefineTableStatement {
 	pub comment: Option<Strand>,
 	#[revision(start = 2)]
 	pub if_not_exists: bool,
+	#[revision(start = 3)]
+	pub table_type: TableType,
```

Comment:
Can we rename this `kind` so it's consistent with `DefineFieldStatement`? I think it's odd having `Table.table_type`...?